lib/amazon-pricing/definitions/instance-type.rb

```diff
@@ -116,6 +116,14 @@ def self.populate_lookups
       end
     end
 
+    def self.disk_bytes_per_sec_capacity(api_name)
+      PER_SEC_CAPACITIES[api_name][0] * 1024 * 1024
```

Comment:
It's megabytes, and it doesn't matter for 2 reasons: first, both have a common 1024 base, and 1000 base in SI, but more importantly, these are values I derived from bytes/s after dividing them by 1024² myself, so 1024 is the right value here.